js/modal.js

```diff
@@ -0,0 +1,101 @@
+const project = {
+  name: 'Keeping track of hundreds of components',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  featured_image: {
+    desktop: 'images/desktop_pop.png',
+    mobile: 'images/mobile_pop.png',
+  },
+  technologies: ['Ruby on rails', 'css', 'JavScript'],
+  live: 'live',
+  source: 'source',
+};
```

Comment:
- [ ] [OPTIONAL] Good job so far! It would be clean if you implements `JavaScript` code to an `array` of `objects` to store JavaScript objects containing the information for each project. Each `object` contains at least the following information about the project: `name`, `description`, `featured image`, `technologies`, `link to live version`, and `link to the project source code`. Ie:

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```